airqo-platform/AirQo-frontend
# [Calibrate] Calibrate App

#### Summary of Changes (What does this PR do?)
- Creates calibrate app

#### Is this change ready to hit production in its current state?

#### Status of maturity (all need to be checked before merging):


#### How should this be manually tested?
* Fetch and checkout to this branch locally
* Creates a `.env` file in the root directory. Contents of the `.env` file
```
REACT_APP_BASE_URL=http://staging-platform.airqo.net/api/v1/
REACT_APP_BASE_CALIBRATE_URL=
NODE_PATH=src/
PORT=5000
```
* Install requirements `npm install`
* Start netmanager staging application `npm run stage`
* Confirm the summary of changes listed above

#### What are the relevant tickets?
- [WEB-134](https://airqoteam.atlassian.net/browse/WEB-134)

#### Screenshots (optional)
